Versie 0.5/SLD Uitwisselberichten/XSDs en voorbeeld berichten/dssad.wsdl

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wat doet dit bestand in de upload naar SLD ?

Reply:
oh wacht, je moet deze hernomen naar dssld.wsdl

Reply:
Done